### DISCUSSION TITLE
Make beatmap info stay on the screen while opening player info overlay

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#22831] [Q&A] [answered]
Beatmap info page should have the same logic as other web overlays.

https://user-images.githubusercontent.com/86934170/224479145-ae1e3d09-6f60-423e-8c69-1b6fddf0de94.mp4

**Comment by Theighlin** [comment, marked as answer]
I think this falls under #5063

**Reply by Loreos7** [reply]
This issue kinda old, abandonded and mixed with other requests. This discussion is direct and particular about one thing.

**Reply by Loreos7** [reply]
https://github.com/ppy/osu/issues/5063#issuecomment-823840830

**Reply by peppy** [reply]
#5063 is still valid, yep.
